src/gui/transferlistsortmodel.cpp

```diff
@@ -29,6 +29,7 @@
 #include "transferlistsortmodel.h"
 
 #include <type_traits>
+#include <concepts>
```

Comment:
```suggestion
#include <concepts>
#include <type_traits>
```

Reply:
To keep them in alphabetical order?

Reply:
>To keep them in alphabetical order?

yes.

Reply:
Happy to change that order. But what about the other inclusions? They are not in alphabetical order either.

> #include "transferlistsortmodel.h"
> 
> #include <concepts>
> #include <type_traits>
> 
> #include <QDateTime>
> 
> #include "base/bittorrent/infohash.h"
> #include "base/bittorrent/torrent.h"
> #include "transferlistmodel.h"

Just for my education and to understand how to do it "the right way" - Is there a reason for that specific order?

Reply:
> Just for my education and to understand how to do it "the right way" - Is there a reason for that specific order?

There are various reasons, some of which are objective, some of which are subjective - there are many discussions on this topic on the web. In general, we have drawn up our own rules based on them:
https://github.com/qbittorrent/qBittorrent/blob/master/CODING_GUIDELINES.md#7-header-inclusion-order

Reply:
Thx for sharing, will study those